CHANGELOG.md

```diff
@@ -77,6 +77,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Bug Fixes
 
+* (baseapp) [#18727](https://github.com/cosmos/cosmos-sdk/pull/18727) Ensure that BaseApp.Init firstly returns any errors from a nil commit multistore instead of panicking on nil dereferencing and before sealing the app.
```

Comment:
```suggestion
* (baseapp) [#18727](https://github.com/cosmos/cosmos-sdk/pull/18727) Ensure that `BaseApp.Init` firstly returns any errors from a nil commit multistore instead of panicking on nil dereferencing and before sealing the app.
```

Reply:
Done, thanks @atheeshp!